### ISSUE TITLE
Error 'Could not find function "appender.file2"' with CRAN version

### ISSUE THREAD

**Issue by DavZim** [zatonovo/futile.logger#65]
If I use the current CRAN-version it appears that the `appender.file2`-function is missing.

When I use the current GitHub-version (using `devtools::intstall_github("zatonovo/futile.logger")`) everything works as expected.

Unfortunately, I am relying on a CRAN-installation, any way around this error?

The error should be reproduced by:

``` r
# install.packages("futile.logger")
library(futile.logger)
packageVersion("futile.logger")
#> [1] '1.4.3'

flog.appender(appender.file2("mylog.log", console=TRUE), name='mylogger')
#> Error in appender.file2("mylog.log", console = TRUE): could not find function "appender.file2"
```

**Comment by leaf-jing** [comment]
file2 is not included in release 1.4.3

**Comment by s-fleck** [comment]
Could we get the current futile.logger version on CRAN? I require the appender2 feature and its easier for me to deploy when its on cran :/

**Comment by aryoda** [comment]
FYI: A CRAN release is in discussion now (no schedule so far, see #98)

**Comment by aryoda** [comment]
> file2 is not included in release 1.4.3

Yes, `appender.file2` was introduced in v1.4.5 which is not yet published at CRAN
(see the [NEWS](https://github.com/zatonovo/futile.logger/blob/master/NEWS.md) file which contains the change history now).
